presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1095,6 +1097,20 @@ public boolean isIgnoreTableBucketing()
         return ignoreTableBucketing;
     }
 
+    @Config("hive.min-bucket-count-to-not-ignore-table-bucketing")
+    @ConfigDescription("If table bucket count is less than this value, table bucketing is always ignored. " +
```

Comment:
nit: `Ignore table bucketing when the number of buckets is less than the value specified`